zulip_bots/zulip_bots/bots/lunch/lunch.py

```diff
@@ -0,0 +1,220 @@
+import random
+
+ORDERS = {}
+
+class LunchBotHandler(object):
+
+    def usage(self):
+        return '''
+        This is a small Zulip bot for organizing lunch with your friends and
+        coworkers.
+        '''
+
+    # MESSAGE RECEIVERS
+    def handle_message(self, message, bot_handler):
+        content = self.keyword_selection(message)
+        print(message)
+        bot_handler.send_reply(message, content)
+
+    def keyword_selection(self, message):
+        msg = message['content'].split()
+
+        if len(msg) == 0:
+            return "I don't think I can do that. :frowning:"
+
+        # Check if a user was referenced
+        user = message['sender_full_name']
+        if msg[0][0] == '@':
+            user = msg[0][1:].replace('**', '')
+            msg  = msg[1:]
+
+        # Find the command
+        if len(msg) == 0:
+            return  # null
+
+        # <user> I want that
+        if msg[-1] == 'that':
+            return copy_user_list(user, message['sender_full_name'])
+
+        # <user|optional> (I) want <item>
+        if msg[0:2] == ['I', 'want']:
+            return add_to_user_list(user, msg[2:])
+
+        if msg[0] == 'wants':
+            return add_to_user_list(user, msg[1:])
+
+        # <user|optional> remove all
+        if msg == ['remove', 'all']:
+            return remove_all(user)
+
+        # <user|optional> remove <item>
+        if msg[0] == 'remove':
+            return remove_from_user_list(user, msg[1:])
+
+        # orders
+        if msg[0] == 'orders':
+            return show_all_orders()
+
+        # <user|optional> order
+        if msg[0] == 'order':
+            return show_order(user)
+
+        # who should <verb>?
+        if msg[0:2] == ['who', 'should']:
+            return random_user(msg[2:])
+
+        if msg[0] == 'help':
+            return '''
+                <user|optional> (I) want(s) <item>
+                Adds <item> to the sender's (or <user>'s if specified) list of items
+
+                <user> I want that
+                Adds every item in <user>'s list of items to the sender's list of items
+
+                <user|optional> remove <item>
+                Removes <item> to the sender's (or <user>'s if specified) list of items
+
+                <user|optional> remove all
+                Removes all items to the sender's (or <user>'s if specified) list of items
+
+                orders
+                Show all orders
+
+                <user|optional> order
+                Shows all items of the sender's (or <user>'s if specified) list of items
+
+                who should <verb>(?)
+                Randomly selects a user from the orders list
+
+                help
+                Shows help (this section)
+            '''
+
+        return "I don't think I can do that. :frowning:"
+
+# User finder
+# Finds if a user exists
+# error is True, return an error message
+# error is False, create the user line
+def find_user(user, error):
```

Comment:
I don't think you need these comments! I think it would be better if you removed them! :)